src/webgl/p5.RendererGL.js

```diff
@@ -383,6 +383,11 @@ p5.RendererGL.prototype.resize = function(w,h) {
   var gl = this.GL;
   p5.Renderer.prototype.resize.call(this, w, h);
   gl.viewport(0, 0, gl.drawingBufferWidth, gl.drawingBufferHeight);
+  // If we're using the default camera, update the aspect ratio
+  if(this._curCamera === 'default') {
+    this._curCamera = null;
```

Comment:
Yes, because _setDefaultCamera checks the value of _curCamera to determine whether or not to do anything. It's one of these functions that you can call liberally, because it only actually executes once (except now we need to override that behavior in this instance).